src/docs/stan-reference/examples.tex

```diff
@@ -8046,6 +8046,154 @@ \section{Circular Representations of Days and Years}
 weekends, or with data normalization back to natural scales for
 daylight savings time.
 
+\chapter{Solving Algebraic Equations}\label{algebra-solver.chapter}
+
+\noindent
+Stan provides a built-in mechanism for specifying and solving systems
+of algebraic equations. The function signatures for Stan's algebraic solver
+can be found in \refchapter{functions-algebraic-solver}.
+%
+Solving any system of algebraic equations can be translated into a root-finding
+problem, that is, given a function $f(x)$, we wish to find $x$ such that
+$f(x) = 0$.
+
+\section{Example: System of Nonlinear Algebraic Equations}
+
+For systems of linear algebraic equations, we recommend solving the system
+using an inverse matrix. The algebraic solver becomes handy when we want
+to solve nonlinear equations.
+%
+As an illustrative example, we consider a nonlinear system of two equations
+with two unknowns. Our goal is to simultaneously solve all equations for
+$x_1$ and $x_2$, such that the vector function $z$ goes to 0. 
```

Comment:
good point. I remember in highschool my math teacher emphasizing the difference between f and f(x).